AdyenCard/Validators/AddressValidator.swift

```diff
@@ -0,0 +1,56 @@
+//
+// Copyright (c) 2022 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal) import Adyen
+import Foundation
+
+/// Validates a card's address field.
+internal class AddressValidator {
+
+    internal func isValid(address: PostalAddress?, addressMode: CardComponent.AddressFormType, addressViewModel: AddressViewModel) -> Bool {
```

Comment:
This could be static?

Reply:
Then I will have to make private func static.